src/Writer/MySQL.php

```diff
@@ -223,6 +225,29 @@ protected function columnNamesForLoad(array $table, array $header): array
         }, $header);
     }
 
+    private function generateColumnVariable(string $columnName): string
+    {
+        $variable = uniqid('@columnVar_');
```

Comment:
Nechces pouzit poradove cislo stlpca? ... bude kratsie a pochopitelnejsie SQL

Reply:
pokud na tom úplně netrváš, tak bych to nechal takto

Reply:
@ondrajodas 

`unique` id ma v dokumentacii nejake varovanie, ze hodnota nemusi byt vzdy unikatna, ... neviem ci to treba riesit, ... ak by to bolo iba niekde v teste tak asi nie, :
https://www.php.net/manual/en/function.uniqid.php

IMHO staci v tomto foreach pouzit index a odovdat ho do volania metody:
https://github.com/keboola/db-writer-mysql/blob/7576121da1a8462849a0c7b24d43110aefa1f221/src/Writer/MySQL.php#L214-L219

Necham to na tvoje rozhodnutie.

Reply:
oki, přehodím to :)